### PR TITLE
Add CIS benchmark scan info to docs

### PR DESCRIPTION
**Pull request by abhay-krishna**
Adding a section in docs for CIS benchmark self-assessment guide using Kube-bench.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/371#" title="Author self-approved">abhay-krishna</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [abhay-krishna]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/content/en/docs/reference/security/kube-bench.md
Comment (chrisnegus) [inline docs/content/en/docs/reference/security/kube-bench.md:33]:
```suggestion
| 4.2.10 | Ensure that the –tls-cert-file and –tls-private-key-file arguments are set as appropriate | When generating serving certificates, functionality could break in conjunction with hostname overrides which are required for certain cloud providers |
```

**Review by chrisnegus — CHANGES_REQUESTED** [review]
Look really good! I just have a few small suggestions.

**Pull request merged**
